apps/discord-bot-internal/src/index.ts

```diff
@@ -0,0 +1,142 @@
+import { Client, Events, GuildMember, Routes } from 'discord.js';
+import { setGlobalDispatcher, Agent } from 'undici';
+import { logger } from './logger';
+import { config } from './config';
+import { replyDescriptionEmbed } from './utils';
+import { MomentumColor } from './momentum-color';
+import { InteractionModule } from './types/interaction-module';
+import { initializeServices } from './services';
+import { interactionModules } from './interaction-modules';
+
+// Some discord api requests are done through undici and require higher timeout
+// https://github.com/nodejs/undici/issues/1531#issuecomment-1178869993
+setGlobalDispatcher(new Agent({ connect: { timeout: 60000 } }));
+
+const client = new Client({ intents: [2 ** 25 - 1] });
+
+const commandMap: Map<string, InteractionModule> = new Map();
+const contextMenus: Map<string, InteractionModule> = new Map();
```

Comment:
Nit (sorry): I really prefer `const commandMap = new Map<string, InteractionModule>();` to this.